homework_37_220423_hibernate/src/main/java/org/example/dao/MovieDao.java

```diff
@@ -0,0 +1,95 @@
+package org.example.dao;
+
+import org.example.entity.Movie;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import util.HibernateUtil;
+
+public class MovieDao {
+
+    public void saveMovie(Movie movie) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Transaction transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            session.close();
+            HibernateUtil.getSessionFactory().close();
```

Comment:
do you really need to close it here?

Reply:
Created separated method closeSessionFactoryConnection()